.github/workflows/run_tests.yml

```diff
@@ -31,12 +31,14 @@ jobs:
 
     - name: Run tests
       shell: bash -l {0}
+      env:
+        KBASE_CI_TOKEN: ${{ secrets.KBASE_CI_TOKEN }}
```

Comment:
current (crappy) test needs to connect to CI. I have some refactoring planned and hope to eliminate the need for a connection once it's done.